.github/workflows/integration.yaml

```diff
@@ -2,7 +2,8 @@ name: Build
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "releases/**" ]
```

Comment:
I've realised using `releases/` as a prefix is a bit of a pain. The docker metadata action normalises it to `releases-` when creating the docker tags. It helps keep things straight if it's an exact match, so maybe our release branch pattern could be something like `rel-**`

Reply:
Why not just call the branch `release-XXXX`?

Reply:
done, the pattern is now `release-**` and instructions are updated. I've tested the release branch flow in my fork too.